File: csharp/ButtplugCSharpFFI/ButtplugHandshakeException.cs
using System;

namespace Buttplug
{
    public class ButtplugHandshakeException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugHandshakeException()
        {
        }

        /// <inheritdoc />
        public ButtplugHandshakeException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugHandshakeException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/exceptions/ButtplugHandshakeException.java
package io.buttplug.exceptions;

public class ButtplugHandshakeException extends ButtplugException {
    public ButtplugHandshakeException(String msg) {
        super(msg);
    }

    public ButtplugHandshakeException(String msg, Exception inner) {
        super(msg, inner);
    }
}
